squirrel_manipulation_msgs/action/Ptp.action

```diff
@@ -0,0 +1,10 @@
+# Manipulation action messages: ptp
+
+# goal definition
+geometry_msgs/Pose pose
```

Comment:
missed the geometry_msgs dependency on the package.xml (https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/squirrel_manipulation_msgs/package.xml) , please add it